android/library/src/main/java/com/ackee/flipper/grpc/FlipperGrpcInterceptor.kt

```diff
@@ -0,0 +1,86 @@
+package com.ackee.flipper.grpc
+
+import com.google.gson.GsonBuilder
+import io.grpc.*
+import java.util.*
+
+/**
+ * Interceptor that logs requests and responses from gRPC server
+ */
+internal class FlipperGrpcInterceptor(
+    private val flipperGrpcPlugin: FlipperGrpcPlugin
+) : ClientInterceptor {
+
+    override fun <ReqT : Any?, RespT : Any?> interceptCall(
+        method: MethodDescriptor<ReqT, RespT>,
+        callOptions: CallOptions,
+        next: Channel
+    ): ClientCall<ReqT, RespT> {
+        val id = UUID.randomUUID().toString()
+
+        return object : ForwardingClientCall.SimpleForwardingClientCall<ReqT, RespT>(
```

Comment:
what about separate class definiton inside the file?

Reply:
c10d37fc064ea28596684833d493db71120b54b3 separated but im not sure if its more readable